assets/sass/components/first-party-mode/_googlesitekit-first-party-mode-setup-cta-banner.scss

```diff
@@ -0,0 +1,50 @@
+/**
+ * First-party mode Setup CTA Banner styles.
+ *
+ * Site Kit by Google, Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.googlesitekit-plugin {
+
+	.googlesitekit-setup-cta-banner--first-party-mode-setup-cta-banner {
+
+		.googlesitekit-setup-cta-banner__actions-wrapper {
+			gap: 0; // Fix buttons being one on top of another on medium mobile viewports.
+
+			@media (min-width: $width-tablet + 1 + px) {
+				gap: $grid-gap-phone;
+			}
+
+			@media (min-width: $width-desktop + 1 + px) {
+				gap: $grid-gap-desktop;
+			}
+		}
```

Comment:
Please update the gaps to be `4px` for the phone viewport and `10px` for tablet/desktop, as per the AC for https://github.com/google/site-kit-wp/issues/9600.